src/types/collaborative/transport_service.ts

```diff
@@ -13,6 +13,7 @@ export interface RemoteRevisionMessage extends AbstractMessage {
   commands: readonly CoreCommand[];
   nextRevisionId: UID;
   serverRevisionId: UID;
+  timestamp?: number;
```

Comment:
I'm actually not sure what it should be.
A real timestamp (number) ? An ISO 8601 string `"2023-05-05T06:56"` ? A `Date` object ?
A string is certainly more readable when debugging, but it's not type safe since it will happily accept the string "prout".
Odoo gives the `create_date` as an ISO string, which can also be taken into consideration

Reply:
I'd say there is no "real better" option. However, the most surprising type for a timestamp amongst string, number or date would be string imo. 
So I'm kinda inclined to keep it as a number.